ethereum/core/src/main/java/org/hyperledger/besu/ethereum/bonsai/BonsaiWorldStateArchive.java

```diff
@@ -255,14 +255,13 @@ public Optional<MutableWorldState> getMutable(final Hash rootHash, final Hash bl
     }
   }
 
-  @Override
-  public MutableWorldState getMutable() {
-    return persistedState;
+  private BonsaiPersistedWorldState persistedState() {
```

Comment:
i'm not sure if this will improve the scenario where two block imports are working at the same time.  BonsaiPersistedWorldState is a gateway to the database, and having two gateways to the same database could introduce a different style of corruption.